### DISCUSSION TITLE
New Formatting Option - comma at beginning of line

### DISCUSSION THREAD

**Discussion by ghost** [TabularEditor/TabularEditor3#399] [Ideas]
Hi, I would very much like an option in the DAX auto formatter to always put the comma separating function arguments at the beginning of new line.

This way it would be very easy for example to comment out/uncomment different FILTER arguments or CALCULATE modifiers.  

What do you think?

**Comment by otykier** [comment]
Hi @Hulkster 
This option is already available when authoring DAX queries in Tabular Editor:

<img width="517" alt="image" src="https://user-images.githubusercontent.com/8976200/152773618-30b28c92-e63e-4040-a490-1480d6f28c86.png">

**Reply by ghost** [reply]
Hi @otykier,
Yes, but it is only available for DAX Queries. Unavailable for Expression Editor where parts of debugging also take place. For example I use Expression Editor together with Performance Analyzer in PowerBI to quickly identify performance impact of different filter combinations. In reevaluates with every measure change.

**Reply by otykier** [reply]
I see. We did not consider this use case. We can absolutely allow formatting with debug commas in the Expression Editor as well. Adding that to the backlog.

**Reply by otykier** [reply]
Debug commas are now available for the regular expression editor as well: [3.2.2](https://github.com/TabularEditor/TabularEditor3/releases/tag/3.2.2).

**Reply by ghost** [reply]
Thanks
